.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: python
+sudo: false
+env:
+  - PYTHON_VERSION=3.5
+before_install:
+  - wget http://repo.continuum.io/miniconda/Miniconda3-3.7.3-Linux-x86_64.sh -O miniconda.sh
+  - chmod +x miniconda.sh
+  - ./miniconda.sh -b
+  - export PATH=/home/travis/miniconda3/bin:$PATH
+  # Update conda itself
+  - conda update --yes conda
+install:
+  - conda create --yes -n env_name python=$PYTHON_VERSION pip numpy scipy nose flake8 h5py
```

Comment:
You'll need to create another environment to install qiita on that environment (see [here](https://github.com/qiita-spots/qiita_client/blob/master/.travis.yml#L14)), especially cause deblur is py3 but Qiita is py2.